util/dockerfiles/gpu-fs/Dockerfile

```diff
@@ -27,7 +27,7 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-FROM --platform=${BUILDPLATFORM} ubuntu:20.04
+FROM ubuntu:20.04
```

Comment:
Why this change? The old version would, in theory, allow this to build Arm binaries as well as x86 binaries.

I know that it's unlikely that someone would want to build Arm binaries for the AMD GPU, but it doesn't hurt to have the old version, from what I understand.

Reply:
Ah ok.  I removed it because I didn't know what it does, but I'll add it back